components/Sections/Team/TeamBlock/index.jsx

```diff
@@ -122,7 +122,7 @@ const TeamBlock = () => {
     },
     {
       name: 'Eugene Tseitlin',
-      position: 'Sr. Technical Writer',
+      position: 'Sr. Full Stack Developer',
```

Comment:
@souravjain540 good catch! please keep the lingo that is already used for the rest of the team
```suggestion
      position: 'Sr. Full-stack Eng.',
```